source/globalCommands.py

```diff
@@ -1463,6 +1463,29 @@ def script_toggleProgressBarOutput(self,gesture):
 	script_toggleProgressBarOutput.__doc__=_("Toggles between beeps, speech, beeps and speech, and off, for reporting progress bar updates")
 	script_toggleProgressBarOutput.category=SCRCAT_SPEECH
 
+	def script_togglePlayObjectCoordinates(self,gesture):

```

Comment:
Could you please change this implementation to follow the implementation strategy of script_braille_toggleFocusContextPresentation? This includes creating constants for objFocus, objNav, both, etc.

Alternatively, you can choose the implementation of script_toggleReportLineIndentation.

Reply:
I agree, perhaps define a dictionary first with the expected values and the ui messages, then you can get the index of the current `playObjectCoordinates` value increment it, and fetch the ui message from the dictionary for the new `playObjextCoordinates` value.